pom.xml

```diff
@@ -341,22 +341,22 @@
       <dependency>
         <groupId>io.thorntail</groupId>
         <artifactId>arquillian</artifactId>
-        <version>2.3.0.Final-SNAPSHOT</version>
```

Comment:
Will this even work when releasing upstream?

I believe we had that issue in the past which is why the versions are explicitly in the pom

Reply:
OK, I'll just revert this part